src/modules/home/components/LoggedInVisitorSpacesHome/LoggedInVisitorSpacesHome.tsx

```diff
@@ -112,7 +112,7 @@ const LoggedInVisitorSpacesHome: FC<DefaultSeoInfo> = ({ url }) => {
 	const { data: visitorSpaceInfo, isError: isErrorGetVisitorSpace } = useGetVisitorSpace(
 		query[QUERY_PARAM_KEY.VISITOR_SPACE_SLUG_QUERY_KEY] as string,
 		false,
-		{ enabled: !!query[QUERY_PARAM_KEY.VISITOR_SPACE_SLUG_QUERY_KEY], retry: false }
+		{ enabled: !!query[QUERY_PARAM_KEY.VISITOR_SPACE_SLUG_QUERY_KEY] }
```

Comment:
`retry: false` seems like a very specific bugfix, can we `git blame` & check for regression?

Reply:
no, i moved the default for that useGetVisitorSpace hook so that retry is false by default
so you no longer have to pass it to the hook explicitly